redash/handlers/organization.py

```diff
@@ -10,13 +12,32 @@
 @login_required
 def organization_status(org_slug=None):
     counters = {
-        "users": models.User.all(current_org).count(),
-        "alerts": models.Alert.all(group_ids=current_user.group_ids).count(),
-        "data_sources": models.DataSource.all(current_org, group_ids=current_user.group_ids).count(),
-        "queries": models.Query.all_queries(current_user.group_ids, current_user.id, include_drafts=True).count(),
-        "dashboards": models.Dashboard.query.filter(
-            models.Dashboard.org == current_org, models.Dashboard.is_archived.is_(False)
-        ).count(),
+        "users": models.db.session.execute(models.User.all(current_org, columns=[func.count(models.User.id)])).first()[
+            0
+        ],
+        "alerts": models.db.session.execute(
+            models.Alert.all(group_ids=current_user.group_ids, columns=[func.count(models.Alert.id)], distinct=[])
```

Comment:
Why do you use `select(func.count(...))` like "dashboards"?

Reply:
do not understand your question

Reply:
updated it to scalar anyway

Reply:
Sorry, I should have described a little more detail... I think that each of these (users / alerts / data_sources / queries / dashboards) gets the number of records that match the conditions.

In the "dashboard" case, the code is below:

```
        "dashboards": models.db.session.scalar(
            select(func.count(models.Dashboard.id)).where(
                models.Dashboard.org == current_org, models.Dashboard.is_archived.is_(False)
            )
        ),
```

However, queries is below:

```
        "queries": models.db.session.scalar(
            models.Query.all(
                current_user.group_ids,
                user_id=current_user.id,
                include_drafts=True,
                columns=[func.count(distinct(models.Query.id))],
            )
        ),
```

Even though both codes get the number of records that meet the conditions, I was concerned that they were written differently. Is there any intention?

I think that it is fine to use "scalar" in these cases 👍

Reply:
Dashboard.all method has more complex conditions with joins, which are not required here

Reply:
Updated dashboards in the same manner as queries